build/assets/tuned/openshift-node-performance

```diff
@@ -59,8 +59,10 @@ kernel.hung_task_timeout_secs = 600
 kernel.nmi_watchdog = 0
 #> realtime
 kernel.sched_rt_runtime_us = -1
-#> cpu-partitioning (= 1) #realtime (= 0)
-kernel.timer_migration = 0
+#> cpu-partitioning and realtime for RHEL disable it (= 0)
+# OCP is too dynamic when partitioning and needs to evacuate
+# scheduled timers when starting a guaranteed workload (= 1)
```

Comment:
I believe that if we want to keep the comments inline we should use the same format as in other place we have here.
meaning - move `>` to the end of the comment section here (line 62 to line 64)

Reply:
make sure to adjust the [testdata/render-expected-output/manual_tuned.yaml](https://github.com/openshift-kni/performance-addon-operators/pull/850/files#diff-853529c02df5410a1582f7d76352236ee3b9de3550af446eba701685027e4156)  if you edit the file as I mentioned above

Reply:
Fixed.